src/style.css

```diff
@@ -0,0 +1,25 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
```

Comment:
[checklist](https://github.com/mate-academy/layout_stars/blob/master/checklist.md) 3rd item. don't hardcode width, let the content inside dictate it
```suggestion
```

Reply:
done